tasks/sep/base-001-MCP-L1/VALIDATION.md

```diff
@@ -0,0 +1,341 @@
+
+Please ensure that each link to the `superchain-registry` correctly corresponds to Sepolia as the superchain registry contains data for
+different chains. The `superchain-registry` is considered the source of truth for contract addresses across the superchain. To ensure
+that the address actually matches the correct implementation, an Etherscan link is also provided for each.
+
+[system-config-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/5ad42cbb49472a0bf164ade976426f7526ee6dfe/superchain/implementations/networks/sepolia.yaml#L14
+[system-config-etherscan]: https://sepolia.etherscan.io/address/0xba2492e52F45651B60B8B38d4Ea5E2390C64Ffb1
+[l1-xdm-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/a62fd38c9239459b27ea50a2df895e89a3305382/superchain/extra/addresses/sepolia/base.json#L3
+[l1-xdm-etherscan]: https://sepolia.etherscan.io/address/0xC34855F4De64F1840e5686e64278da901e261f20
+[l1-erc721-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/a62fd38c9239459b27ea50a2df895e89a3305382/superchain/extra/addresses/sepolia/base.json#L4
+[l1-erc721-etherscan]: https://sepolia.etherscan.io/address/0x21eFD066e581FA55Ef105170Cc04d74386a09190
+[portal-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/a62fd38c9239459b27ea50a2df895e89a3305382/superchain/extra/addresses/sepolia/base.json#L8
+[portal-etherscan]: https://sepolia.etherscan.io/address/0x49f53e41452C74589E85cA1677426Ba426459e85
+[batch-inbox-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/5ad42cbb49472a0bf164ade976426f7526ee6dfe/superchain/configs/sepolia/metal.yaml#L9
+[batch-inbox-etherscan]: https://sepolia.etherscan.io/address/0x24567B64a86A4c966655fba6502a93dFb701E316
+[l1-standard-bridge-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/a62fd38c9239459b27ea50a2df895e89a3305382/superchain/extra/addresses/sepolia/base.json#L5
+[l1-standard-bridge-etherscan]: https://sepolia.etherscan.io/address/0xfd0Bf71F60660E2f608ed56e1659C450eB113120
+[factory-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/a62fd38c9239459b27ea50a2df895e89a3305382/superchain/extra/addresses/sepolia/base.json#L7
+[factory-etherscan]: https://sepolia.etherscan.io/address/0xb1efB9650aD6d0CC1ed3Ac4a0B7f1D5732696D37
+[output-oracle-registry]: https://github.com/ethereum-optimism/superchain-registry/blob/a62fd38c9239459b27ea50a2df895e89a3305382/superchain/extra/addresses/sepolia/base.json#L6
+[output-oracle-etherscan]: https://sepolia.etherscan.io/address/0x84457ca9D0163FbC4bbfe4Dfbb20ba46e48DF254
+
+todo: this looks like an initialization but it wasn't present in the template
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000002` <br/>
+  **After:** `0x0000000000000000000000000000000000000000000000000000000000000001` <br/>
+  **Meaning:** 
```

Comment:
placeholder TODO for this + any other applicalble reinitializer changes that we need to explain